tailwind.config.js

```diff
@@ -24,23 +24,23 @@ module.exports = {
       h3: ["var(--fs-h3)", "var(--lh-min-normal)"],
       h4: ["var(--fs-h4)", "var(--lh-min-normal)"],
       h5: ["var(--fs-h5)", "var(--lh-min-normal)"],
-      p1: ["var(--fs-p1)", "var(--lh-extra-relaxed)"],
-      p2: ["var(--fs-p2)", "var(--lh-extra-relaxed)"],
-      p3: ["var(--fs-p3)", "var(--lh-extra-relaxed)"],
-      p4: ["var(--fs-p4)", "var(--lh-extra-relaxed)"],
+      p1: ["var(--fs-p1)", "var(--lh-relaxed)"],
+      p2: ["var(--fs-p2)", "var(--lh-relaxed)"],
+      p3: ["var(--fs-p3)", "var(--lh-relaxed)"],
+      p4: ["var(--fs-p4)", "var(--lh-relaxed)"],
       standfirst: ["var(--fs-standfirst)"],
       "standfirst-xl": ["var(--fs-standfirst-xl)"],
       "sub-header": ["var(--fs-sub-header)"],
       "sub-header-xs": ["var(--fs-sub-header-xs)"],
       overline1: ["var(--fs-overline1)"],
       overline2: ["var(--fs-overline2)"],
-      label1: ["var(--fs-label1)"],
-      label2: ["var(--fs-label2)"],
-      label3: ["var(--fs-label3)"],
-      label4: ["var(--fs-label4)"],
+      label1: ["var(--fs-label1)", "var(--lh-snug)"],
```

Comment:
This is the improvement to the consolidated styles, setting the line height via the TW config